sdk/storage/azure-storage-file-datalake/tests/perfstress_tests/append.py

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import uuid
+
+from azure_devtools.perfstress_tests import RandomStream, AsyncRandomStream
+
+from ._test_base import _FileSystemTest
+
+
+class AppendTest(_FileSystemTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        file_name = "filetest-" + str(uuid.uuid4())
```

Comment:
Each instance should append to its own file

Reply:
It is - this constructor is scoped to the instance, not global. And the `setup` method constructs the file specific to this instance.